### DISCUSSION TITLE
Socket io on server with nginx

### DISCUSSION THREAD

**Discussion by PGCodehub** [miguelgrinberg/python-socketio#1073] [Q&A] [answered]
Hi @miguelgrinberg  

Thanks for creating the library 

I have a few questions on how to connect socket io with properly Nginx 

My context here is I am running python socket-io on the server, I have other API endpoints running here and I am running Nginx to proxy to endpoints, and using guicorn eventlet for socket io

`gunicorn -k eventlet -w 1 app:app --reload`

 How can we change port 8000 that socket io running?
   
 Thank you for the help

**Comment by miguelgrinberg** [comment, marked as answer]
There is an example nginx configuration in the Flask-SocketIO documentation: https://flask-socketio.readthedocs.io/en/latest/deployment.html#using-nginx-as-a-websocket-reverse-proxy.
